qiskit_braket_provider/providers/braket_job.py

```diff
@@ -77,26 +78,26 @@ def _get_result_from_aws_tasks(
     return experiment_results
 
 
-class AWSBraketJob(JobV1):
-    """AWSBraketJob."""
+class AmazonBraketTask(JobV1):
+    """AmazonBraketTask."""
 
     def __init__(
         self,
-        job_id: str,
+        task_id: str,
         backend: BackendV2,
         tasks: Union[List[LocalQuantumTask], List[AwsQuantumTask]],
-        **metadata: Optional[dict]
+        **metadata: Optional[dict],
     ):
-        """AWSBraketJob for local execution of circuits.
+        """AmazonBraketTask for local execution of circuits.
 
         Args:
-            job_id: id of the job
+            task_id: id of the task
             backend: Local simulator
             tasks: Executed tasks
             **metadata:
         """
-        super().__init__(backend=backend, job_id=job_id, metadata=metadata)
-        self._job_id = job_id
+        super().__init__(backend=backend, task_id=task_id, metadata=metadata)
+        self._task_id = task_id
```

Comment:
Could we add a method to access the `task_id` similar to the one we have in the JobV1 class?

Reply:
I have added this part in the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/d4e6f0c6136113dff66804592c03dd763ef1697a